lib/bluesky.js

```diff
@@ -21,7 +20,7 @@ let bluesky = {};
   };
 
   bluesky.endSession = function (caps) {
-    logger.info("BlueSky: End Session");
+    logger.info(`BlueSky: End Session`);
```

Comment:
Whats up with the quotes?

Reply:
Just standardizing with the ` and ${}, as using + uses concatenation and is slower.